x-pack/test/functional_with_es_ssl/fixtures/plugins/alerts/public/application.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, RouteComponentProps } from 'react-router-dom';
+import { EuiPage, EuiText } from '@elastic/eui';
+import { AppMountParameters, CoreStart } from '../../../../../../../src/core/public';
+
+export interface AlertingExampleComponentParams {
```

Comment:
I wonder if it would be kosher-ish for us to reuse the example alert plugin here; would ensure we keep the example up-to-date!  And cut down on yet another fixture.

Reply:
I don't think so - The fixture is prebaked with test data and behaviour while example is about providing users with real world examples of how to use this.